prac_05/emails.py

```diff
@@ -0,0 +1,31 @@
+"""
+Emails
+Estimate: 30 mins
+Actual: 40 mins
+"""
+
+
+def main():
+    email_to_names = {}
+    email = input("Email: ")
+    while email != "":
+        full_name = extract_name(email)
+        confirm = input(f"Is your name {full_name}? (y/n)").lower()
```

Comment:
Perhaps a different name for `confirm` could work better. This variables essentially acts to "**confirm** the users **name**"